src/main/interface/cli.c

```diff
@@ -3463,8 +3473,8 @@ static void printResource(uint8_t dumpMask)
         }
 
         for (int index = 0; index < MAX_RESOURCE_INDEX(resourceTable[i].maxIndex); index++) {
-            const ioTag_t ioTag = *((const ioTag_t *)currentConfig + resourceTable[i].offset + index);
-            const ioTag_t ioTagDefault = *((const ioTag_t *)defaultConfig + resourceTable[i].offset + index);
+            const ioTag_t ioTag = *((const ioTag_t *)currentConfig + resourceTable[i].stride * index + resourceTable[i].offset);
```

Comment:
Indexing is done on value of type `ioTag_t`, but `uint8_t` is correct (stride and offset is in bytes)